xcm/pallet-xcm/Cargo.toml

```diff
@@ -42,5 +42,5 @@ std = [
 ]
 runtime-benchmarks = [
 	"frame-system/runtime-benchmarks",
-	"xcm-builder/runtime-benchmarks",
+	"xcm-builder",
```

Comment:
```suggestion
```

This is the proper solution.

Reply:
I see. so its not needed for benchmarks as well